src/main/java/com/ezasm/parsing/Line.java

```diff
@@ -115,6 +116,15 @@ public String getLabel() {
         return label;
     }
 
+    /**
+     * Get the Stored Register Number
+     *
+     * @return the Stored Register Number
+     */
+    public int getRegisterNum() {
```

Comment:
This method, and the `registerNum` variable in general seem to not be used at all. Consider removing